styles.css

```diff
@@ -1,3 +1,3 @@
 .my-greeting {
-    background-color: cyan;
+    background-color: #FA8072;
 }
```

Comment:
![third hello microverse linters](https://user-images.githubusercontent.com/93039359/176288114-a91b02c2-e81b-44ab-9b5c-86ffeeed8e75.png)
- [x] You have set up your linters great :+1: But your linters are not passing all the tests. Please check your errors and kindly fix them. It might be an extra line you need to add at the very end of your CSS file. :+1: